.github/workflows/updatecli.yml

```diff
@@ -0,0 +1,44 @@
+---
+name: "Updatecli: Dependency Management"
+
+on:
+  workflow_dispatch:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: '15 */2 * * *'
+  repository_dispatch:
+    types: [kubewarden-controller-release]
```

Comment:
I'm not sure this is having any effect, as I don't think we have a `kubewarden-controller-release`. In any case, I think it would be better to trigger on the github.com/kubewarden/helm-charts releases.

Reply:
Yeah, I agree. I would remove this and add a trigger to run the job manually. Then, if we decided to automate this, we can update the helm chart workflow to trigger this job afterwards. Or just leave this CI running once a day.

Reply:
How can it be triggered after the helm chart workflow?

Reply:
We can do it later after some testing. The idea is to mark this workflow as `workflow_dispatch` ([example](https://github.com/kubewarden/helm-charts/blob/main/.github/workflows/update-charts.yml#L4)), and then configure other repository, such as kubewarden/helm-charts, to trigger this one that we are adding on this PR. [Example of that trigger](https://github.com/kubewarden/kubewarden-controller/blob/main/.github/workflows/release.yml#L162-L168). The repository and client payload would be different.